python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -117,13 +117,19 @@ def run(self, patchRef, selectDataList=[]):
             except (KeyError, ValueError):
                 visitId = i
 
-            exp = self.createTempExp(calexpRefList, skyInfo, visitId)
-            if exp is not None:
+            exps = self.createTempExp(calexpRefList, skyInfo, visitId)
+
+            if exps.exposure is not None or exps.exposurePsfMatched is not None:
                 dataRefList.append(tempExpRef)
-                if self.config.doWrite:
-                    self.writeCoaddOutput(tempExpRef, exp, "tempExp")
             else:
                 self.log.warn("tempExp %s could not be created", tempExpRef.dataId)
+
+            if self.config.doWrite:
+                if exps.exposure:
+                    self.writeCoaddOutput(tempExpRef, exps.exposure, suffix="tempExp")
```

Comment:
I think this was a temporary state with with the old struct names: exposure and psfMatchedExposure.